cmd/build/remote/build.go

```diff
@@ -64,8 +66,8 @@ func (bc *OktetoBuilder) Build(ctx context.Context, options *types.BuildOptions)
 		options.File = filepath.Join(path, "Dockerfile")
 	}
 
-	if err := utils.CheckIfRegularFile(options.File); err != nil {
-		return fmt.Errorf("%s: %s", oktetoErrors.InvalidDockerfile, err.Error())
+	if exists := filesystem.FileExistsAndNotDir(options.File, afero.NewOsFs()); !exists {
+		return fmt.Errorf("%s: is not a regular file", oktetoErrors.InvalidDockerfile)
```

Comment:
we could be returning another error with this change. Is this change safe? Maybe the customer who consumes it acts on the error, that's my concern

Reply:
adding this wrapping actually makes the error returned the same as before when using `CheckIfRegularFile`, if you see the return from that old func:

```go
fmt.Errorf("'%s' is not a regular file", path)
```

What differs is that if `fs.Stat` fails, we return a different error but I think it's ok.